source/app.d

```diff
@@ -33,7 +33,7 @@ void startMonitoring()
 		sleep(1.seconds()); // give the cache a chance to warm up first
 		while(true){
 			if (s_mirror.length) s_registry.mirrorRegistry(s_mirror);
-			else s_registry.updatePackages();
```

Comment:
uhh you do realize this code is the same as the production code and you removed package updates with this?

Reply:
Yes of course I do. That's why it's called experiment / WIP. It crashes sometimes when trying to update the registry while wget/curl is fetching. I just wanted to see how far I can get with a simple curl loop.

Anyhow, I can/will remove this once `--no-monitoring` is a thing (-> https://github.com/dlang/dub-registry/pull/310)